README.md

```diff
@@ -40,17 +40,17 @@ Run "start.sh" with the following parameters:
 
 ./start.sh -t Test -p 6000 -c ~/Data/1/
 
-./start.sh -t Test -p 6000 -j '{"version":1, "formation_name":"433-433"}'
+./start.sh -t Test -p 6000 -j '{"version":1, "formation_name":"433"}'
 
-./start.sh -t Test -p 6000 -j '{"version":1, "formation_name":"433-433"}' -e json
+./start.sh -t Test -p 6000 -j '{"version":1, "formation_name":"433"}' -e json
 
-./start.sh -t Test -p 6000 -j #q#{#qq#version#qq#:1#c##qq#formation_name#qq#:#qq#433-433#qq#}#q# -e temp
+./start.sh -t Test -p 6000 -j #q#{#qq#version#qq#:1#c##qq#formation_name#qq#:#qq#433#qq#}#q# -e temp
 ```
 
 ### Json Settings
-- `formation_name`: name of formation (default: "433-433") options "4231-3133", "433-343l", "433-433", "442-334", "442-424", "523-5232"
-- `winner_formation_name`: name of formation when our score is more than opponent score (default: "433-433") options "4231-3133", "433-343l", "433-433", "442-334", "442-424", "523-5232"
-- `loser_formation_name`: name of formation when opponent score is more than our score (default: "433-433") options "4231-3133", "433-343l", "433-433", "442-334", "442-424", "523-5232"
+- `formation_name`: name of formation (default: "433") options "4231-3133", "433-343l", "433", "442-334", "442-424", "523-5232"
```

Comment:
formation_name should be "formation" in the readme and in code

winner and loser should be removed from readme, but you don't need to remove their code from base

Reply:
resolved